code/datums/ai_laws.dm

```diff
@@ -117,6 +126,21 @@ var/global/const/base_law_type = /datum/ai_laws/nanotrasen
 		zeroth_law_borg = null
 	sorted_laws.Cut()
 
+/datum/ai_laws/proc/set_sixsixsix_law(var/law)

```

Comment:
Perhaps make the devil law a subtype of a Zeroth law, where /zeroth/devil/return_index() returns 666. It means it's not technically a Zeroth law, but it'd be unremovable.